packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -16,6 +16,9 @@ import {
 } from 'react-native';
 import { NativeModules } from 'react-native';
 
+const BatchedBridge = require('BatchedBridge');
```

Comment:
These are haste imports, which will stop working in 0.61.

NativeEventEmitter should be imported from react-native:
import {NativeEventEmitter} from 'react-native';

BatchedBridge doesn't appear to be exported from react-native and requires a path import:
import BatchedBridge from 'react-native/Libraries/BatchedBridge/BatchedBridge';

Reply:
I copied this code base on what @vmoroz had in other samples. I've tried removing the BatchedBridge but then the app just throws up a JS warning every time the event fires, and the JS callback doesn't execute.

Reply:
TLDR: I'm updating the samples to not use BatchedBridge. While it is possible and supported to create your own event emitters - no one does it for native modules.